repository/SmalltalkCI-Pharo-Core.package/SCIPharoCodeCoverage.class/class/coverallsBuildData.st

```diff
@@ -2,7 +2,8 @@ compatibility
 coverallsBuildData
 	| reference |
 	reference := self coverallsBuildDataFilename asFileReference.
-	[ ^ (reference readStreamDo: [ :stream | STON fromStream: stream ]) ]
-		on: STONReaderError do: [ :error |
-			error messageText: 'Cannot read ', self coverallsDataFilename, '.'.
+	[ ^ reference readStreamDo: [ :stream | STON fromStream: stream ] ]
+		on: STONReaderError
+		do: [ :error | 
+			error messageText: 'Cannot read ' , self coverallsBuildDataFilename , '.'.
```

Comment:
This is unrelated but fixes a possible DNU

Reply:
Thanks!